core/org.wso2.carbon.user.core/src/main/java/org/wso2/carbon/user/core/common/AbstractGroupManagementErrorEventListener.java

```diff
@@ -176,4 +176,75 @@ public boolean onListGroupsFailure(String errorCode, String errorMessage, Condit
 
         return true;
     }
+
+    @Override
+    public boolean onAddGroupFailure(String errorCode, String errorMessage, String groupName, List<String> userIDs,
+                                     List<Claim> claims, UserStoreManager userStoreManager)
+            throws UserStoreException {
+
+        return true;
+    }
+
+    @Override
+    public boolean onAddGroupFailure(String errorCode, String errorMessage, List<String> userIDs,
+                                     Group group, UserStoreManager userStoreManager) throws UserStoreException {
```

Comment:
Why we have two `onAddGroupFailure` methods, one with group name and other one with group object?